.changeset/lazy-comics-swim.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/polaris': patch
+---
+
+Added experimental `xl` and `2xl` sizes
+Reintroduced `round` shape
```

Comment:
> Reintroduced `round` shape

Not sure we need to mention this. In my previous PR to update Avatar I was only overriding the styles behind our flag. I never removed this prop.